dump-persistence-schema-compiler-plugin/src/main/scala/org/virtuslab/ash/writer/PersistenceSchemaWriter.scala

```diff
@@ -11,14 +11,14 @@ import org.virtuslab.ash.model.TypeDefinition
 class PersistenceSchemaWriter(outputDirectory: File) extends DumpPersistenceSchemaJsonProtocol {
 
   def this(dumpPersistenceSchemaOptions: DumpPersistenceSchemaOptions) = {
-    this(File(dumpPersistenceSchemaOptions.outputDir))
+    this(File(dumpPersistenceSchemaOptions.outputDir) / "dump-persistent-schema-cache")
```

Comment:
Actually, I can't get dump schema to work on my `chore/ash-single-sbt-plugin` branch... I'm now using `ashDumpPersistenceSchema` task (see `ci/dump-akka-events-json`, yet unnamed), and I'm getting:

```
$ ./ci/dump-akka-events-json target/akka-persistence-schema.yaml
.....
[success] Total time: 18 s, completed 16 Sep 2021, 11:33:13

$ cat target/akka-persistence-schema.yaml
[]
```

Pls take a look 🕵🏻

Reply:
```scala
Compile / ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginEnable := false,
Test / ashDumpPersistenceSchemaCompilerPlugin / ashCompilerPluginEnable := true
```
Seems some leftovers from testing were the issue :sweat_smile: